index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const pump = require('pump');
+const {pipeline} = require('stream');
```

Comment:
Reading `pipeline` at the callsite is not clear enough what it actually is. Should either be `stream.pipeline` or `streamPipeline`.

Reply:
I changed to `streamPipelinePromisified` since I'm using promisify now

Reply:
You prefer to rename it on require time?
```js
const {pipeline:streamPipeline} = require('stream')

const streamPipelinePromisified = promisify(streamPipeline)
```

Reply:
You can just import it as `const stream = require('stream');` and then `const streamPipelinePromisified = promisify(stream.pipeline)`